Emmanuelaaron/todoList
# # Writing tests

In this pull request, I've written tests for Creating the projects and the todos using jest.

# how to test this PR
- clone this repo
- checkout to the ```testing``` branch
- ```npm install```
- Run ```npm test```
- All tests should be passing
